src/components/Button.jsx

```diff
@@ -0,0 +1,19 @@
+import { useNavigate } from "react-router-dom";
+import styles from "./Button.module.css";
+
+//styleType은 small, medium, large로 나뉜다
+function Button ({ styleType="small", className, onClick, to, children, ...props }) {
```

Comment:
❗️ 수정요청
className이 없을 때 undefined가 들어갈 수 있으므로 기본값을 빈 문자열로 해주시는 것을 추천드려요!
```suggestion
function Button ({ styleType="small", className = '', onClick, to, children, ...props }) {
```